lib/pure/os.nim

```diff
@@ -296,6 +296,10 @@ proc setCurrentDir*(newDir: string) {.inline, tags: [].} =
   else:
     if chdir(newDir) != 0'i32: raiseOSError(osLastError())
 
+when defined(posix):
+   var
+     pathMax {.importc: "PATH_MAX", header: "<stdlib.h>".}: cint
```

Comment:
Isn't this already declared elsewhere?

Reply:
Is it? I couldn't find any reference to it. Do you know where?

I just brought this back from an earlier commit which was reverted later, but maybe now this definition is not necessary anymore